console/src/main/java/org/jline/console/impl/DefaultPrinter.java

```diff
@@ -512,15 +511,17 @@ private Object mapValue(Map<String, Object> options, String key, Map<String,Obje
     @SuppressWarnings("unchecked")
     private List<String> optionList(String key, Map<String,Object> options) {
         List<String> out = new ArrayList<>();
-        if (options.containsKey(key)) {
-            if (options.get(key) instanceof String) {
-                out.addAll(Arrays.asList(((String)options.get(key)).split(",")));
-            } else if (options.get(key) instanceof Collection) {
-                out.addAll((Collection<String>)options.get(key));
-            } else {
-                throw new IllegalArgumentException("Unsupported option list: {key: " + key
-                                                 + ", type: " + options.get(key).getClass() + "}");
-            }
+        Object option = options.get(key);
+        if (option == null) {
+            return out;
+        }
+        if (option instanceof String) {
+            out.addAll(Arrays.asList(((String)option).split(",")));
+        } else if (option instanceof Collection) {
+            out.addAll((Collection<String>)option);
+        } else {
+            throw new IllegalArgumentException("Unsupported option list: {key: " + key
+                                             + ", type: " + option.getClass() + "}");
         }
```

Comment:
It would be nicer to inline the first check in the last one using:
```
} else if (option != null) {
    throw new IllegalArgumentException(...
```